### PR TITLE
KH-157: Move 'Other' concept to general.csv file

### PR DESCRIPTION
**Pull request by kdaud**
Refer to [KH-157](https://issues.openmrs.org/browse/KH-157)

### CODE REVIEW

File: configs/openmrs_config/concepts/ethnicities.csv [hunk 1]
```diff
@@ -28,5 +28,4 @@ bac9af94-07b3-44f2-b678-5f02ad5959e4,,,Robel,រ៉ូបែល,Robel,រ៉ូ
 8d57672b-bc62-49dd-a417-ef62b8522275,,,Poang,ប៉ូអាង?,Poang,ប៉ូអាង?,,,Misc,Text,,,,
 769d258d-c6f8-477e-a5ed-10ff07c80fe3,,,Roong,រ៉ូអ៊ុង,Roong,រ៉ូអ៊ុង,,,Misc,Text,,,,
 57fe19cb-c83e-4778-99f6-a20d55b40191,,,Other Indigenous,ជនជាតិដើមភាគតិចផ្សេងទៀត,Other Indigenous,ជនជាតិដើមភាគតិចផ្សេងទៀត,,,Misc,Text,,,,
-5622AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,,,Other,ផ្សេងទៀត,Other,ផ្សេងទៀត,,,Misc,Text,,,,
```
Comment (kazlaw) [inline configs/openmrs_config/concepts/ethnicities.csv:31]: @kdaud, did you update the Uuid for all occurences of this in the forms to match the new Uuid for `Other`

**Review by kazlaw — COMMENTED** [review]
@kdaud  is this a typo in the heading `H-157: Move 'Other' concept to general.csv file` ?

File: configs/openmrs_config/concepts/ethnicities.csv (hunk 1 above)
Comment (kdaud) [reply configs/openmrs_config/concepts/ethnicities.csv:31]: The uuid hasn't changed but rather refactoring `Other` concept from being duplicated.

**Comment by kdaud** [comment]
@kazlaw have addressed the reviews.

**Review by rbuisson — CHANGES_REQUESTED** [review]
For this to work, we must make sure that general.csv is processed before ethnicities.csv, otherwise the latter will fail.
@kdaud , could you set the `_order` in ethnicities.csv to be `1100`?

**Pull request merged**